author	John Stultz <john.stultz@linaro.org>	2017-01-03 22:40:13 -0800
committer	John Stultz <john.stultz@linaro.org>	2017-05-12 14:09:26 -0700
commit	7c27abcb9687786bc4a05885c308cc0a22b909ba (patch)
tree	175188552e1f1d7874c3b08e826f2591164d4728
parent	5e3530082e5adf009a867364b76f9db013e49dc9 (diff)
download	flo-7c27abcb9687786bc4a05885c308cc0a22b909ba.tar.gz
HACK: add mac address
Re-add mac address bits

Signed-off-by: John Stultz <john.stultz@linaro.org>
Diffstat
-rw-r--r--	arch/arm/boot/dts/qcom-apq8064.dtsi	1	
		
1 files changed, 1 insertions, 0 deletions
diff --git a/arch/arm/boot/dts/qcom-apq8064.dtsi b/arch/arm/boot/dts/qcom-apq8064.dtsi
index f3db185..0fae486 100644
--- a/arch/arm/boot/dts/qcom-apq8064.dtsi
+++ b/arch/arm/boot/dts/qcom-apq8064.dtsi
@@ -1528,6 +1528,7 @@
 
 						qcom,smem-states = <&apps_smsm 10>, <&apps_smsm 9>;
 						qcom,smem-state-names = "tx-enable", "tx-rings-empty";
+						local-mac-address = [ 18 00 2d 88 9c a9 ];
 					};
 				};
 			};
